src/pages/VerifySettingsPage/VerifySettingsPage.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState, useCallback } from "react";
+import { useDeskproAppEvents, useDeskproAppClient } from "@deskpro/app-sdk";
+import { getAccounts } from "../../api/api";
+import { VerifySettings } from "../../components/VerifySettings";
+import type { FC } from "react";
+import type { Maybe, Settings } from "../../types";
+import type { IAccount } from "../../api/types";
+
+const getError = (err: Maybe<Error>): string => {
```

Comment:
This shouldn't use `Maybe` as we always get an error passed in here as `.catch((err) => setError(getError(err)))` will ensure that.

Reply:
fixed